test/functionsSimulation.test.ts

```diff
@@ -17,50 +17,62 @@ describe("Test chainlink:sandbox:functionsSimulation module [SKIP FOR GITHUB ACT
       const result =
```

Comment:
not sure we need this extra abstraction on the functions toolkit simulator and a test -- neither abstraction nor test add anything to the plugin.

Reply:
@zeuslawyer that's true, hardhat plugin is just a thin wrapper around functions-toolkit, as it was requested and it is by design.
Is it necessary to remove the simulator functionality?

Reply:
thanks @koteld - you're right, the mandate is a thin wrapper.  Let's leave as is then.